backend/internal/sys/config/conf.go

```diff
@@ -26,8 +26,13 @@ type Config struct {
 }
 
 type Options struct {
-	AllowRegistration    bool `yaml:"disable_registration" conf:"default:true"`
-	AutoIncrementAssetID bool `yaml:"auto_increment_asset_id" conf:"default:true"`
+	AllowRegistration       bool `yaml:"disable_registration" conf:"default:true"`
+	AutoIncrementAssetID    bool `yaml:"auto_increment_asset_id" conf:"default:true"`
+	HeaderSSOEnabled        bool `yaml:"header_sso_enabled" conf:"default:false"`
+	HeaderSSOAllowedIP      string `yaml:"header_sso_allowed_ip" conf:"default:0.0.0.0"`
+	HeaderSSOAutoRegister   bool `yaml:"header_sso_autoregister" conf:"default:true"`
+	HeaderSSOHeaderName     string `yaml:"header_sso_header_name" conf:"default:Remote-Name"`
+	HeaderSSOHeaderEmail    string `yaml:"header_sso_header_email" conf:"default:Remote-Email"`
```

Comment:
Do we need to do any kind of validation to make sure that these values are valid? If possible it would be nice to prevent to user from using "bad" values and we can just log the error and exit. 

not super necessary, but can be nice in avoiding issues with mis-configuration.

Reply:
Hm, right now I can't think of any meaningful validation being done here. I thought about `HeaderSSOAllowedIP` but I am not sure if it always has to be an actual IP address. So I renamed that `HeaderSSOAllowedAddr`.
If you have ideas what could be validated here, I will be happy to add.